CMakeLists.txt

```diff
@@ -17,8 +17,13 @@ set(HEADERS
   "src/strbuf.h"
 )
 
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
 add_executable(clinfo "${SOURCE_FILE}" "${HEADERS}")
 
+target_link_libraries(clinfo ${CONAN_LIBS})
```

Comment:
Are you sure this step is necessary? I tested it and even with this step my system (which also works with a non-Conan build) picked up the system OpenCL instead of the Conan-provided one.